gcp-lib.js

```diff
@@ -0,0 +1,60 @@
+/* eslint no-use-before-define: 0 */
+
+const _ = require("lodash");
+const GCCompute = require("@google-cloud/compute");
+const helpers = require("./helpers");
+
+const RESOURCE_OPERATIONS = {
+  create: _.partial(callMethod, "insert"),
+  get: async (client, request) => (await _.partial(callMethod, "get")(client, request))[0],
+  delete: _.partial(callMethod, "delete"),
+  listAsync: _.partial(callMethod, "listAsync"),
+  searchProjectsAsync: _.partial(callMethod, "searchProjectsAsync"),
```

Comment:
Absolutely no negative thing to say about this, just wanted to say that I love this design and the usage of partials here.